html_of_wiki.opam

```diff
@@ -1,8 +1,8 @@
 # This file is generated by dune, edit dune-project instead
 opam-version: "2.0"
-synopsis: "a static website generator for software projects"
+synopsis: "A wikicreole to HTML compiler"
 description:
-  "a static website generator for software projects, using wikicreole syntax"
+  "html_of_wiki is a static website generator used by the Ocsigen project to manage its online documentation."
```

Comment:
keep the original: "for software project". It is not supposed to be for Ocsigen only.

Reply:
Perhaps the opam file has been regenerated from dune-project and dune-project should be updated instead to avoid this change?

Reply:
Yes it's that. html_of_wiki.opam was previously changed, instead of changing dune-project. I will update dune-project too, so html_of_wiki will be generated with the correct description.